packages/core/upload/server/controllers/admin-folder-file.js

```diff
@@ -40,10 +40,12 @@ module.exports = {
 
     if (deletedFiles.length + deletedFolders.length > 1) {
       strapi.telemetry.send('didBulkDeleteMediaLibraryElements', {
-        rootFolderNumber: deletedFolders.length,
-        rootAssetNumber: deletedFiles.length,
-        totalFolderNumber,
-        totalAssetNumber: totalFileNumber + deletedFiles.length,
+        groupProperties: {
```

Comment:
I think that, based on @fdel-car comment, you've made the change from `groupProperties` to `eventProperties`, but only on **L234** and not on **L43**